src/devices/Mcp960X/Samples/README.md

```diff
@@ -0,0 +1,2 @@
+## Samples
```

Comment:
please move this README.md to the main README.md in the folder above - we're currently in the process of merging those

Reply:
README.md from Samples directory merged with README.m from folder above